files/sync.sh

```diff
@@ -18,15 +18,10 @@ log_error_exit() {
   exit 1
 }
 
-# Create non-root user
-if [ "$UNISON_USER" != "root" ]; then
-  log_heading "Setting up non-root user ${UNISON_USER}."
-  HOME="/home/${UNISON_USER}"
-  addgroup -g $UNISON_GID -S $UNISON_GROUP
-  adduser -u $UNISON_UID -D -S -G $UNISON_GROUP $UNISON_USER
-  mkdir -p ${HOME}/.unison
-  chown -R ${UNISON_USER}:${UNISON_GROUP} ${HOME}
-fi
+export UNISON_UID=$(id -u)
+export UNISON_GID=$(id -g)
+log_heading "Setting up HOME for user uid ${UNISON_UID}."
+sudo chown -R ${UNISON_UID}:${UNISON_GID} ${HOME} ${SYNC_DESTINATION}
```

Comment:
I believe the current user is `root` when this line is executed.  This means `${HOME}` evaluates to `/root`.  Is your intent really to change the permissions on the `/root` folder?  Also, because the current user is `root`, the `sudo` can be eliminated.